### ISSUE TITLE
accelsearch frequency range

### ISSUE THREAD

**Issue by tanumanghosh** [scottransom/presto#171]
Hi,

I am running accelsearch within a frequency range of 0.01 to 6.8 Hz by restricting -flo and -fhi. However, it is searching between 0.0 to 6.8 Hz! I am using -numharm 1, so it should search within the specified range only, right? However, the range of Fourier frequency bins is showing correctly, I guess. Is it just a printing issue?

**Comment by scottransom** [comment]
It might be at least partially a printing issue, but you need to be careful about using `-flo` and `-fhi` with harmonic summing as described in the FAQ here:
https://github.com/scottransom/presto/blob/master/FAQ.md#ive-restricted-the-frequency-range-of-my-search-using--flo-fhi-or--rlo-rhi-but-accelsearch-is-still-returning-candidates-outside-of-that-range-why-is-that

**Comment by tanumanghosh** [comment]
Just to be sure, if I use -numharm 1, it is a single harmonic search. In this case, the frequency range should be strictly between -flo and -fhi. Am I understanding it correctly?

**Comment by scottransom** [comment]
Correct.  But that is certainly not the best way to search for radio pulsars!

**Comment by tanumanghosh** [comment]
I am working with X-ray pulsars. I am searching with different -numharm values. I just wanted to make sure that the range which is printed for "f" in accelsearch is a typo or not! Thank you.
